### ISSUE TITLE
[🐞]: Antique Ink cannot be used on non-directional Signs

### ISSUE THREAD

**Issue by sapphiriza** [MehVahdJukaar/Supplementaries#1062]
### Before Continuing:

- [X] Check you are using the latest version of the mods and its dependencies
- [X] Always include a latest.log if you are crashing
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version

1.20.1-FABRIC

### Supplementaries Version

Supplementaries-1.20-2.8.10

### Moonlight Lib Version

1.20-2.11.33

### Issue Detail

Attempting to use the Antique Ink on a sign will instead open the GUI for editing the sign text. It does work on directional signposts on fences, but does not work on vanilla signs or hanging signs.

**Comment by MehVahdJukaar** [comment]
this is more of amissing feature. antique ink is mostly a forge only feature as it relies on forge block entity capabilityes. Might be a while until i look into this

**Comment by sapphiriza** [comment]
Oh ok, it's weird that it _does_ work on the directional signs... seems like it's just that the edit-sign-contents GUI is interrupting the antique ink interaction?

**Comment by MehVahdJukaar** [comment]
By directional signs you mean vanilla floor signs?

**Comment by VitaminJake** [comment]
No he means the ones that attach to a signpost, which I had assumed were added by supple. I also have a very vested interest in this.
![Base Profile Screenshot 2024 06 12 - 07 50 46 57](https://github.com/MehVahdJukaar/Supplementaries/assets/162518177/5fbb6543-3902-4e5a-884e-ab8d17221d02)

**Comment by MehVahdJukaar** [comment]
yeah those are called way signs and of course they wokr as they are added by the mod and dont rely on attatched tile entity capabilities like normal ones do. As ive said i might look into ways to add that for fabric but thats low priority now
